core/kazoo_attachments/src/kz_att_http.erl

```diff
@@ -116,7 +119,15 @@ handle_fetch_attachment_resp({'ok', <<"\r\n", _/binary>>=Multipart}
     lager:debug("recv multipart response, looking for attachment"),
     handle_multipart_fetch(Multipart, AttSettings, Routines);
 handle_fetch_attachment_resp({'ok', Body}, #{'base64_encode_data' := 'true'}, _Routines) ->
-    {'ok', base64:decode(Body)};
+    try base64:decode(Body) of
+        Decoded ->
+            lager:debug("decoded base64 response"),
+            {'ok', Decoded}
+    catch
+        'error':_R ->
+            lager:debug("response was not base64 encoded"),
+            {'ok', Body}
+    end;
```

Comment:
i think this is not needed, the `httpc:request` called from kz_http does the decoding if needed

Reply:
I don't believe this to be true. At least, when I updated the testing in pqc_cb_storage I was getting the encoded results back before I patched this. After this change I get the raw MP3 back. Might be because of the pqc_httpd not sending headers that httpc needs to auto-decode but we'll definitely have servers that are incomplete in this way.

Reply:
actually, there's no protocol handling for base64-encoded binary as the body of a http request (no header that can be set on response to indicate it). to use base64-encoded binary, one would have to use mime encoding (event with only one part) and provider Content-Transfer-Encoding in that part. so the pqc_httpd needs to decode the parts on post/put and return the binary image as-is on get.

Reply:
i would prefer to support well behaved http servers and this could be a blocker (compat issues later) when we start to stream the attachment instead of return the all binary in memory.